src/components/password-modal/password-modal-component.tsx

```diff
@@ -194,28 +192,25 @@ export class PasswordModalComponent extends React.Component<
     }
 
     private clearErrorMessage() {
-        this.setState({ errorMessage: EMPTY_STRING });
+        this.setState({ errorMessage: '' });
     }
 
     private handlePasswordAttempts() {
+        const failedLoginBlocking = Object.keys(FAILED_LOGIN_BLOCKING)
+            .concat(String(RESET_APP_FAILED_LOGINS))
+            .reverse();
```

Comment:
to be sure the array is ordered descending you should use: `.sort((a,b) => b-a)`